instrumentation/log4j/log4j-context-data/log4j-context-data-common/testing/src/main/java/io/opentelemetry/instrumentation/log4j/contextdata/Log4j2Test.java

```diff
@@ -30,6 +30,10 @@ boolean expectBaggage() {
     return false;
   }
 
+  boolean expectLoggingKeys() {
```

Comment:
what do you think instead having a method
```java
String getLoggingKey(String key) {
  return key;
}
```
and using it like `getLoggingKey("trace_id")`
In the logging keys test this method could return `key + "_test"`